### PR TITLE
merge dev-calvin-vision 

### PR DESCRIPTION
**Pull request by CalSch**

### CODE REVIEW

**Review by KeithRieck — APPROVED** [review]
Looks good.

File: src/main/java/frc/robot/subsystems/CameraSubsystem.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  UsbCamera camera1;
+  UsbCamera camera2;
+  /** Creates a new CameraSubsystem. */
+  public CameraSubsystem() {
+    camera1 = CameraServer.startAutomaticCapture(0);
+    camera1.setResolution(320, 240);
+    camera1.setFPS(7);
+
+    camera2 = CameraServer.startAutomaticCapture(1);
+    camera1.setResolution(320, 240);
```
Comment (KeithRieck) [inline src/main/java/frc/robot/subsystems/CameraSubsystem.java:21]: Should these lines change `camera2` rather than `camera1` ?

**Review by KeithRieck — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/CameraSubsystem.java (hunk 1 above)
Comment (CalSch) [reply src/main/java/frc/robot/subsystems/CameraSubsystem.java]: yes, thanks!

**Review by KeithRieck — APPROVED** [review]
Looks good

**Review by beardedone55 — APPROVED** [review]
LGTM

**Pull request merged**